modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/app/TessellationIOApp.scala

```diff
@@ -253,6 +255,15 @@ abstract class TessellationIOApp[A <: CliMethod](
       }
     }
 
+  private def jarHash(currentEnv: AppEnvironment): IO[Hash] =
+    JarSignature
+      .jarHash[IO]
+      .onError(e => logger.error(s"Error calculating jar's hash: ${e.getMessage}."))
```

Comment:
It's better to pass error explicitly, as it records the stack trace too:
`logger.error(e)("Error calculating jar's hash")`